package.json

```diff
@@ -19,6 +19,7 @@
     "react-slick": "^0.28.1",
     "slick-carousel": "^1.8.1",
     "styled-components": "^5.3.0",
+    "truncate-zero": "^1.0.0",
```

Comment:
highly doubt you need an npm package. please uninstall and write the function yourself